nsxt/resource_nsxt_policy_context_profile_custom_fqdn.go

```diff
@@ -0,0 +1,182 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/vsphere-automation-sdk-go/lib/vapi/std/errors"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/protocol/client"
+	gm_infra "github.com/vmware/vsphere-automation-sdk-go/services/nsxt-gm/global_infra/context_profiles/custom_attributes"
+	gm_model "github.com/vmware/vsphere-automation-sdk-go/services/nsxt-gm/model"
+	infra "github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra/context_profiles/custom_attributes"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+)
+
+func resourceNsxtPolicyContextProfileCustomFQDN() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtPolicyContextProfileCustomFQDNCreate,
+		Read:   resourceNsxtPolicyContextProfileCustomFQDNRead,
+		Delete: resourceNsxtPolicyContextProfileCustomFQDNDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"fqdn": {
```

Comment:
Should we allow to configure CUSTOM_URL as part of same resource?

Reply:
Yeah we could rename this resource to "custom_attribute" and have both. Would it make this more usable?